ui/server/src/main/scala/pl/touk/nussknacker/ui/api/WebResources.scala

```diff
@@ -11,7 +11,8 @@ class WebResources(publicPath: String) extends Directives {
   private lazy val mainContentFile = {
     val tempMainContentFile = Files.createTempFile("nussknacker", "main.html").toFile
     tempMainContentFile.deleteOnExit()
-    val content = IOUtils.toString(getClass.getClassLoader.getResourceAsStream("web/static/main.html"))
+    val data = getClass.getClassLoader.getResourceAsStream("web/static/main.html")
+    val content = Option(data).map(IOUtils.toString).getOrElse("")
```

Comment:
Can you write a comment that this fallback is in case if you hasn't got client app in resources and want to at least has working backend. Am I right that it is that case? Maybe we could also add some logging for this case because someone can spend a while figuring out what is wrong when he/she skip fe preparation.

Reply:
I added log.error - outside development this really should not happen